homeassistant/components/velbus/__init__.py

```diff
@@ -111,7 +89,7 @@ def set_memo_text(service):
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
     """Remove the velbus connection."""
     unload_ok = await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
-    hass.data[DOMAIN][entry.entry_id]["cntrl"].stop()
+    await hass.data[DOMAIN][entry.entry_id]["cntrl"].stop()
```

Comment:
Remove leftover index
```suggestion
    await hass.data[DOMAIN][entry.entry_id].stop()
```